pkg/codegen/templates/wrappers.tmpl

```diff
@@ -25,9 +25,11 @@ func (w *ServerInterfaceWrapper) {{.OperationId}} (ctx echo.Context) error {
 {{end}}
 {{end}}
 
+{{if .SecurityDefinitions}}
```

Comment:
I don't see how this changes anything.

Previously, if security definitions was nil or empty, we would range over nothing, producing no code. This seems like a no-op

Reply:
Oh, I see. I read the related issue.

Could we be much more explicit about this? Rater than inferring global vs local security schemes, can we have a flag instead? This way, you can switch on the value of that flag in the template code - it's easier to read and clearer.

Reply:
I guess it would be clearer if we decide beforehand in the Go-Code calling the template, which `SecurityDefinitions` "win", since we have everything in our hands. This way the template can stay dumb enough and either attached its own definitions or gets a copy of the global definitions.

Reply:
That sounds good.